packages/sdk/src/router/v6/router.ts

```diff
@@ -360,7 +366,7 @@ export class Router {
     const zeroexV4Erc721Details: ListingDetailsExtracted[] = [];
     const zeroexV4Erc1155Details: ListingDetailsExtracted[] = [];
     const zoraDetails: ListingDetailsExtracted[] = [];
-
+    const elementDetails: ListingDetailsExtracted[] = [];
```

Comment:
Similar to ZeroExV4 we could have combinations of ERC721 and ERC1155 fills so probably worth having separate details/executions for these.